jormungandr-lib/src/interfaces/config/node.rs

```diff
@@ -100,9 +100,9 @@ pub struct P2p {
     /// the p2p discovery from.
     pub trusted_peers: Vec<TrustedPeer>,
 
-    /// Listen address as a multiaddr.
+    /// Listen address.
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub listen_address: Option<SocketAddr>,
+    pub listen: Option<SocketAddr>,
```

Comment:
I got it as well, maybe we forgot to update that (and only updated https://github.com/input-output-hk/jormungandr/blob/ba946989dba164f4918d44ab561deacf8b71b66a/jormungandr/src/settings/start/config.rs#L65) because this field was never used in tests so no errors popped up